README.md

```diff
@@ -206,17 +182,13 @@ Firstly you need to open the browser at the link <http://localhost:8091> and edi
 Zwave settings:
 
 - **Serial port**: The serial port where your controller is connected
-- **Network key** (Optional): Zwave network key if security is enabled. The correct format is `"0xCA,0xFE,0xBA,0xBE,.... "` (16 bytes total)
-- **Logging**: Enable/Disable Openzwave Library logging
-- **Save configuration**: Store zwave configuration in `zwcfg_<homeHex>.xml` and `zwscene.xml` files this is needed for persistent node information like node name and location
-- **Poll interval**: Interval in milliseconds between polls (should not be less than 1s per device)
+- **Network key** (Optional): Zwave network key if security is enabled. The correct format is `"CAFEBABE.... "` (32 chars total)
+- **Log level**: Set the zwave-js Library log level
+- **Log to file**: Enable this to store zwave-js logs to a file
 - **Commands timeout**: Seconds to wait before automatically stop inclusion/exclusion
-- **Configuration Path**: The path to Openzwave devices config db
-- **Assume Awake**: Assume Devices that support the Wakeup Class are awake when starting up OZW
-- **Auto Update Config File**: Auto update Zwave devices database
 - **Hidden settings**: advanced settings not visible to the user interface, you can edit these by setting in the settings.json
   - `zwave.plugin` defines a js script that will be included with the `this` context of the zwave client, for example you could set this to `hack` and include a `hack.js` in the root of the app with `module.exports = zw => {zw.client.on("scan complete", () => console.log("scan complete")}`
-  - `zwave.options` overrides options passed to the zwave client see `IConstructorParameters` in [the open-zwave docs](https://github.com/OpenZWave/node-openzwave-shared/blob/master/types/openzwave-shared.d.ts). For detail for example `zwave.options.options.EnforceSecureReception=true` to drop insecure messages from devices that should be secure.
+  - `zwave.options` overrides options passed to the zwave client see `IConstructorParameters` in [the open-zwave docs](https://github.com/zwave-js/node-zwave-js-shared/blob/master/types/zwave-js-shared.d.ts). For detail for example `zwave.options.options.EnforceSecureReception=true` to drop insecure messages from devices that should be secure.
```

Comment:
Text still mentions ozw

Reply:
I think that just replacing `openzwave` with `zwave-js` here is not correct :)

Reply:
docs still need some work like hass and scenes, I just started with them :)